src/tokenizer/state.js

```diff
@@ -6,20 +6,15 @@ import { types as tt } from "./types";
 
 export default class State {
   init(options: Object, input: string) {
-    this.strict = options.strictMode === false ? false : options.sourceType === "module";
+    this.strict = options.strictMode === false
+      ? false
+      : options.sourceType === "module";
 
     this.input = input;
 
     this.potentialArrowAt = -1;
 
-    this.inMethod =
-      this.inFunction =
-      this.inGenerator =
-      this.inAsync =
-      this.inPropertyName =
-      this.inType =
-      this.noAnonFunctionType =
-        false;
+    this.inMethod = this.inFunction = this.inGenerator = this.inAsync = this.inPropertyName = this.inType = this.noAnonFunctionType = false;
```

Comment:
feel free to add `// @prettier-ignore` just before if you want to keep formatting for some special cases.